examples/TO_MIGRATE/ramses/sedo_amr.py

```
@@ -0,0 +1,255 @@
+        return (0.0, 0.0, 0.0)
+
+    model.set_field_value_lambda_f64("rho", rho_map)
+    model.set_field_value_lambda_f64("rhoetot", rhoe_map)
+    model.set_field_value_lambda_f64_3("rhovel", rhovel_map)
+
+    # tmax = 0.03
+    tmax = t_final
+    fact = 15
+    all_t = np.linspace(0, tmax, fact)
+
+    def plot(t, iplot):
```

Ruff (0.16.2)

[warning] Missing return type annotation for private function `plot`

Add return type annotation: `None`

(ANN202)